.github/workflows/build.yml

```diff
@@ -28,6 +44,25 @@ jobs:
         ls -aFlrt
         echo " "
 
+    #! -------------------------------------------------------------------------
+    - name: Update PATH
+      run: |
+        # This is the $PATH reported by `brew install` for the new binaries
+        if [ "$RUNNER_OS" == "macOS" ]; then
+          echo "/opt/homebrew/opt/llvm/bin" >> $GITHUB_PATH
```

Comment:
We need to do this up-front as the rest of the tests that fire-up `l3_dump.py` will need to find and run `llvm-readelf`.